project-base/app/config/graphql/types/ModelType/Slider/SliderItem.types.yaml

```
@@ -23,6 +23,15 @@ SliderItem:
             gtmCreative:
                 type: "String"
                 description: "GTM creative"
+            description:
+                type: "String!"
+                description: "Slider description"
+            rgbBackgroundColor:
+                type: "String"
+                description: "RGB color code for the background of the slider description box"
+            opacity:
+                type: "Float"
```

```suggestion
                type: "Float!"
```
the `opacity` is non nullable on the entity